config/windows-containerd-monitor-filelog.json

```diff
@@ -18,8 +18,18 @@
 		},
 		{
 			"type": "temporary",
-			"reason": "IncompatibleContainer",
-			"pattern": ".*CreateComputeSystem.*"
+			"reason": "ContainerProcessStartFailed",
+			"pattern": ".*failed to start.*hcsshim::System::CreateProcess.*The system cannot find the file specified.*"
```

Comment:
Does this match if the customer tries to run a container with an entry point of a binary that doesn't exist?

Reply:
Is this the same as attempting to deploy a container that does not exist? If so, the error that I see is this: 
`time="2021-04-28T18:06:11.499594200Z" level=error msg="PullImage \"gcr.io/michelletamdua\" failed" error="failed to pull and unpack image \"gcr.io/michelletamdua:latest\": failed to resolve reference \"gcr.io/michelletamdua:latest\": unexpected status code [manifests latest]: 400 Bad Request"`

Here I attempted to deploy an non existent image called gcr.io/michelletamdua to the windows node.

Reply:
```Dockerfile
FROM mcr.microsoft.com/windows/nanoserver:1809
ENTRYPOINT ["/does-not-exist-lol.exe"]
```

Deploy an image like that what do you see?

Reply:
I see this error 
`time="2021-04-29T19:01:41.047663300Z" level=error msg="StartContainer for \"50e3cda4fd5d6100ab71e8b55f49a31b20afc50030e2af61986dc85868c165ee\" failed" error="failed to start containerd task \"50e3cda4fd5d6100ab71e8b55f49a31b20afc50030e2af61986dc85868c165ee\": hcsshim::System::CreateProcess 50e3cda4fd5d6100ab71e8b55f49a31b20afc50030e2af61986dc85868c165ee: The system cannot find the file specified.\n(extra info: {\"CommandLine\":\"/does-not-exist-lol.exe\",\"User\":\"ContainerUser\",\"WorkingDirectory\":\"C:\\\\\",\"Environment\":{\"KUBERNETES_PORT\":\"tcp://10.0.0.1:443\",\"KUBERNETES_PORT_443_TCP\":\"tcp://10.0.0.1:443\",\"KUBERNETES_PORT_443_TCP_ADDR\":\"10.0.0.1\",\"KUBERNETES_PORT_443_TCP_PORT\":\"443\",\"KUBERNETES_PORT_443_TCP_PROTO\":\"tcp\",\"KUBERNETES_SERVICE_HOST\":\"10.0.0.1\",\"KUBERNETES_SERVICE_PORT\":\"443\",\"KUBERNETES_SERVICE_PORT_HTTPS\":\"443\"},\"CreateStdOutPipe\":true,\"CreateStdErrPipe\":true}): unknown"
time="2021-04-29T19:01:41.347059100Z" level=info msg="RemoveContainer for \"4bef6ef5458daf929e13dac1fc6b5ee571097d0c71d34378488bbc5e0d2e7c93\""
time="2021-04-29T19:01:41.370855400Z" level=info msg="RemoveContainer for \"4bef6ef5458daf929e13dac1fc6b5ee571097d0c71d34378488bbc5e0d2e7c93\" returns successfully"`

Reply:
so looks like it produces the same error

Reply:
If the user can accidentally trigger this error with a bad image then I'd remove this condition.